soc/arm/st_stm32/stm32g0/Kconfig.defconfig.stm32g0b1xx

```diff
@@ -11,4 +11,7 @@ config SOC
 config NUM_IRQS
 	default 31
 
+config SHARED_INTERRUPTS
+	default y if $(dt_nodelabel_enabled,fdcan1) && $(dt_nodelabel_enabled,fdcan2)
+
```

Comment:
This should also depends on CAN. Sorry for the late finding.

Reply:
So just changing it to:

```
config SHARED_INTERRUPTS
        select CAN
	default y if $(dt_nodelabel_enabled,fdcan1) && $(dt_nodelabel_enabled,fdcan2)
```

 ?